interop/wasi/wasi_interop_test.go

```diff
@@ -0,0 +1,96 @@
+// package wasi_interop tests wazero's WASI wasi_snapshot_preview1 is compatible with at least one runtime.
+// Links:
+//    - Spec: https://github.com/WebAssembly/WASI/blob/snapshot-01/phases/snapshot/docs.md
+//    - Witx: https://github.com/WebAssembly/WASI/blob/snapshot-01/phases/snapshot/witx/wasi_snapshot_preview1.witx
+//
+// We use tinygo, to perform these tests as the maintainers are familiar with it.
+//
+// Note: Use `make build.interop` to compile `testdata/*.wasm` from `testdata/*.go`
+// Note: This also substitutes for WASI spec tests until we have another option
+package wasi_interop
```

Comment:
In general, we avoid using multi word (or hyphened words) in package name (see https://go.dev/blog/package-names)
```suggestion
package wasi
```

Reply:
Fixed. Thanks.